core/src/main/java/bio/terra/pearl/core/service/notification/email/EnrolleeEmailService.java

```diff
@@ -80,8 +80,9 @@ public void processNotification(Notification notification, Trigger config, Enrol
             // the notification might have been saved, but in a transaction not-yet completed (if, for example,
             // study enrollment transaction is taking a long time). So retry the update if it fails
             RetryTemplate retryTemplate = RetryTemplate.defaultInstance();
-            FixedBackOffPolicy backOffPolicy = new FixedBackOffPolicy();
-            backOffPolicy.setBackOffPeriod(2000);  // this will retry once every two seconds for 3 tries
+            ExponentialBackOffPolicy backOffPolicy = new ExponentialBackOffPolicy();
+            backOffPolicy.setInitialInterval(2000);
```

Comment:
this could use a comment explaining what will happen.  it will retry after intervals of 2, 4, 8, 16, and 32 seconds?  I might also make the initial interval longer -- maybe 5 seconds (or 8 if we want powers of 2)